### PR TITLE
[NDS-707] TextInputBase [v5]

### PR DESCRIPTION
**Pull request by kostasdano**
## Description

[NDS-707](https://orfium.atlassian.net/browse/NDS-707)

- Update TextInputBase props (props changes details can be found inside the jira ticket)
- Update all components that render TextInputBase with the new props.
- Update TextInputBase styles (new tokens).
- Delete obsolete files.
- Update stories (mostly props).
- Update snapshots.

[NDS-707]: https://orfium.atlassian.net/browse/NDS-707?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Select/Select.style.ts [hunk 1]
```diff
@@ -14,11 +14,10 @@ export const selectWrapper =
...
 
-export const rightIconContainer =
-  (isOpen: boolean, isSearchable: boolean) => (): SerializedStyles =>
-    css`
-      display: flex;
-      cursor: pointer;
-      transform: rotate(${isOpen && !isSearchable ? '180' : '0'}deg);
-      ${transition(0.2)}
-    `;
+export const suffixContainer = (isOpen: boolean, isSearchable: boolean) => (): SerializedStyles =>
```
Comment (panvourtsis) [inline src/components/Select/Select.style.ts:17]: suffix/prefix containers can be one for all Field components? if this is one can we move it to shareable folder? what do you think?

**Comment by panvourtsis** [comment]
also UI tests and anything chromatic related should be skipped till parent branch goes into PR for next. Until then chromatic is not stable as it gets them as new components.

File: src/components/Filter/components/SearchInput/SearchInput.tsx [hunk 2]
```diff
@@ -32,13 +32,12 @@ const SearchInput = ({ onChange, value, dataTestId, isLoading }: SearchInputProp
     <div css={textFieldWrapper()(theme)}>
       <TextField
         autoFocus
-        styleType={'filled'}
         onChange={onChange}
         data-testid={generateTestDataId('filter-input', dataTestId)}
-        status="normal"
+        status={{ type: 'normal' }}
```
[the remark this answers is not in the document]
Comment (kostasdano) [reply src/components/Filter/components/SearchInput/SearchInput.tsx:37]: I think memoizing the component with a custom equality check function such as isEqual by lodash would be pretty efficient and rerender preventive

[the remark this answers is not in the document]
Comment (panvourtsis) [reply src/components/Filter/components/SearchInput/SearchInput.tsx:37]: yes it would

File: src/components/Select/Select.style.ts (hunk 1 above)
Comment (panvourtsis) [reply src/components/Select/Select.style.ts:17]: can we add them into the scope of the other ticket though before proceeding? 🙏

**Review by panvourtsis — APPROVED** [review]

**Pull request merged**